### ISSUE TITLE
cannot compile LKRG (Linux Kernel Runtime Guard) with Qubes dom0 kernel / broken gcc plugins structleak_plugin.so latent_entropy_plugin.so

### ISSUE THREAD

**Issue by adrelanos** [QubesOS/qubes-issues#5456]
**Qubes OS version**
R4

**Affected component(s) or functionality**
debian-10 template, likely any template

**Brief summary**
LKRG cannot be compiled

**To Reproduce**
download https://www.openwall.com/lkrg/
extract
cd ~/lkrg-0.7
make

**Expected behavior**
Successful compilation.

**Actual behavior**
```
user@debian-buster-standalone:~/lkrg-0.7$ make
make -C /lib/modules/4.19.79-1.pvops.qubes.x86_64/build M=/home/user/lkrg-0.7 modules
make[1]: Entering directory '/lib/modules/4.19.79-1.pvops.qubes.x86_64/build'
  CC [M]  /home/user/lkrg-0.7/src/modules/ksyms/p_resolve_ksym.o
cc1: error: cannot load plugin ./scripts/gcc-plugins/structleak_plugin.so
   ./scripts/gcc-plugins/structleak_plugin.so: undefined symbol: _ZN11symtab_node10dump_tableEP8_IO_FILE
cc1: error: cannot load plugin ./scripts/gcc-plugins/latent_entropy_plugin.so
   ./scripts/gcc-plugins/latent_entropy_plugin.so: undefined symbol: _Z15build_decl_statj9tree_codeP9tree_nodeS1_
make[2]: *** [scripts/Makefile.build:304: /home/user/lkrg-0.7/src/modules/ksyms/p_resolve_ksym.o] Error 1
make[1]: *** [Makefile:1519: _module_/home/user/lkrg-0.7] Error 2
make[1]: Leaving directory '/lib/modules/4.19.79-1.pvops.qubes.x86_64/build'
make: *** [Makefile:91: all] Error 2
```

**Solutions you've tried**
Use Qubes VM kernel. Works.

**Comment by marmarek** [comment]
Looks like a regression of #2844 . But note the "solution" there was disabling gcc plugins for out-of-tree module builds, which may or may not be desirable in your case.

>  Use Qubes VM kernel. Works.

This is generally better solution, in line of #5212

**Comment by andrewdavidwong** [comment]
> Looks like a regression of #2844 .

I'm interpreting this to mean that this issue should be closed as a duplicate of #2844, and #2844 should be re-opened due to the regression. (If this is incorrect, please let me know and/or feel free to undo these actions.)

**Comment by andrewdavidwong** [comment]
Duplicate of #2844

**Comment by marmarek** [comment]
The issue was the fix was only in a master branch (kernel-latest-* packages), but not relevant stable branches. Fixed (4.19.82, 4.14.152).
